docs/getting-started/data.md

```diff
@@ -254,6 +254,7 @@ export default function Root() {
 With the route configured, we can create an [`action`][action]. Actions are like [loaders][loader] except instead "reading" data they "write" data. Think of it like `React.useState`. It returns a reader and a writer. In React Router, loaders are your readers, actions are your writers.
 
 ```jsx
+// Sample code - this does not actually work :)
```

Comment:
Do you think this is useful to be very clear this code doesn't actually work like this?